src/audio/pcm_converter/pcm_converter_generic.c

```diff
@@ -136,6 +138,63 @@ static void pcm_convert_s32_to_s24(const struct audio_stream *source,
 
 #endif /* CONFIG_FORMAT_S24LE && CONFIG_FORMAT_S32LE */
 
+#if CONFIG_FORMAT_FLOAT && (CONFIG_FORMAT_S16LE || CONFIG_FORMAT_S24LE || CONFIG_FORMAT_S32LE)
+
+static inline int32_t _pcm_shift(int32_t d, int32_t a)
```

Comment:
An inline comment explaining what this does is needed here.

Reply:
added